lib/elementWrapper/checkBoxWrapper.js

```diff
@@ -51,8 +51,17 @@ function getCheckBoxElementWithLabel(searchElement, label) {
  * @extends {ElementWrapper}
  */
 class CheckBoxWrapper extends ElementWrapper {
-  constructor(attrValuePairs, _options, ...args) {
+  constructor(attrValuePairs, dollarSelector, _options, ...args) {
```

Comment:
Can we make it generic so that other selectors like radiobutton, dropdown etc can also accept a `$` selector ?